README.md

```diff
@@ -119,6 +154,16 @@ Additionally, for the section pages and for posts to have a fallback image, add
 default_og_image = "static/ocean.jpg"
 ```
 
+### Mastodon
+
+To enable [Mastodon verification](https://docs.joinmastodon.org/user/profile/#verification), add
+the following to your `config.toml`:
+
+```toml
+[extra]
+mastodon_url = "https://hachyderm.io/@opeik"
```

Comment:
Might be good to "anonymize" the URL e.g.

`mastodon_url = "https://mastodon.server/@yourusername`